common/command.cc

```diff
@@ -181,6 +181,9 @@ po::options_description CommandHandler::getGeneralOptions()
     general.add_options()("placed-svg", po::value<std::string>(), "write render of placement to SVG file");
     general.add_options()("routed-svg", po::value<std::string>(), "write render of routing to SVG file");
 
+    general.add_options()("timing-report", po::value<std::string>(),
+                          "write timing analysis report in CSV format to file");
```

Comment:
looks like this is doing JSON, not CSV

Reply:
Right, I forgot to change that in the help string.